tensorflow/compiler/xla/service/hlo_verifier.cc

```diff
@@ -2702,10 +2702,17 @@ class InstructionVerifier : public DfsHloVisitorWithDefault {
         });
     // TODO(b/259609697): Support FP8 operands in all instructions that support
     // inputs of other floating-point dtypes. Currently the CPU and GPU backends
-    // only support FP8 operands in the convert instruction.
-    if (has_fp8_operand && instruction->opcode() != HloOpcode::kConvert) {
+    // only support FP8 operands in the convert, tuple, get-tuple-element and
+    // transpose instructions and FP8 Custom Calls.
+    if (has_fp8_operand && instruction->opcode() != HloOpcode::kConvert &&
+        instruction->opcode() != HloOpcode::kTuple &&
+        instruction->opcode() != HloOpcode::kGetTupleElement &&
+        instruction->opcode() != HloOpcode::kTranspose &&
+        !(instruction->opcode() == HloOpcode::kCustomCall &&
+          instruction->custom_call_target() == "__cublas$lt$matmul$f8")) {
```

Comment:
Is there a reason to not allow other custom calls to use FP8?

Reply:
This particular custom call has been tested to work with FP8 operands. Additional custom calls can be added after they have been verified to work with FP8.

Reply:
In my mind custom calls is open to extension. We can't say there won't be other custom calls that allow fp8 types written out-of-tree. In my opinion we should support that.

Reply:
This PR only intends to support cuBLAS gemms, not to allow FP8 in arbitrary custom calls. But I suppose it couldn't hurt to allow arbitrary custom calls in this PR instead of a new PR. @philipphack can you remove the `custom_call_target` check?